Common/FeatureFlags.swift

```diff
@@ -90,13 +91,18 @@ struct FeatureFlagConfiguration: Decodable {
         #else
         self.includeServicesInSettingsEnabled = true
         #endif
-        
-        // Swift compiler config is inverse, since the default state is enabled.
+
         #if MOCK_THERAPY_SETTINGS_ENABLED
         self.mockTherapySettingsEnabled = true
         #else
         self.mockTherapySettingsEnabled = false
         #endif
+        
+        #if DELETE_ALL_BUTTON_ENABLED
```

Comment:
Sorry, I guess I wasn't clear here. The default behavior should be if the environment variable is NOT defined (for DIY Loop). Therefore, since we want to disable the Delete All button in Tidepool Loop, the environment variable needs to be called `DELETE_ALL_BUTTON_DISABLED` while the property is `deleteAllButtonEnabled`.

Think about it this way, for DIY Loop, *none* of these environment variables will be defined. Therefore, with the current implementation `deleteAllButtonEnabled` will be `false`.

So, in short, rename `DELETE_ALL_BUTTON_ENABLED` to `DELETE_ALL_BUTTON_DISABLED`, leave `deleteAllButtonEnabled` named the same, but reverse the `true` and `false` below. Finally, add the "// Swift compiler config is inverse, since the default state is enabled." comment above this line.